tests/conftest.py

```diff
@@ -1480,8 +1480,10 @@ def fixture_context_config(
             "root": str(mip_dir),
         },
         "rnafusion": {
-            "binary_path": "/path/to/bin",
+            "binary_path": "/path/to/bin/nextflow",
```

Comment:
```suggestion
            "binary_path": Path("path", "to", "bin", "nextflow").as_poisix,
```
Would that work? It is usually good too avoid hard coding "/" and let Python handle the OS separator via pathlib.